.gitignore

```diff
@@ -29,3 +29,4 @@ out
 .gradle
 test-results/
 licences\.iml
+licence-specs/src/*.iml
```

Comment:
this is fine - an alternative would be to merge this with line above with `**/*.iml`